openapi/components/schemas/Company.yaml

```
@@ -0,0 +1,49 @@
+type: object
+description: |-
+  Company information associated with the customer primary email address domain.
```

nitpick: ( we can use [possessive apostrophes](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#possessive-apostrophes) for people)
```suggestion
  Company information that is associated with the customer's primary email address domain.
```